src/g1.rs

```diff
@@ -418,6 +424,25 @@ impl G1Affine {
     }
 }
 
+/// A nontrivial third root of unity in Fp
+pub const BETA: Fp = Fp([
+    0x30f1_361b_798a_64e8,
+    0xf3b8_ddab_7ece_5a2a,
+    0x16a8_ca3a_c615_77f7,
+    0xc26a_2ff8_74fd_029b,
+    0x3636_b766_6070_1c6e,
+    0x051b_a4ab_241b_6160,
+]);
```

Comment:
This code could be added to the file:
```
p=4002409555221667393417789825735904156556882819939007885332058136124031650490837864442687629129015664037894272559787
Fp = GF(p)
Fpx.<x> = Fp[]

β = 793479390729215512621379701633421447060886740281060493010456487427281649075476305620758731620350
R = 1<<384
tmp = ZZ(Fp(β*R))
while tmp > 0:
    print("0x{:_x}, ".format(tmp % (1<<64)))
    tmp >>= 64
```
The choice in [this issue](https://github.com/zkcrypto/bls12_381/pull/80) is the opposite. This can be verified by changing β to -β in the previous code.

Reply:
Thanks @simonmasson, I added this sagemath code to `notes/design.rs`

I also added another line, `β == (Fp.multiplicative_generator() ** ((p-1)/3))`, which definitively computes β as a nontrivial third root of unity.

There should not be any functional difference between these two choices of BETA.